lnbits/settings.py

```diff
@@ -292,6 +292,9 @@ class EnvSettings(LNbitsSettings):
     lnbits_commit: str = Field(default="unknown")
     super_user: str = Field(default="")
     version: str = Field(default="0.0.0")
+    login_secret: str = Field(
```

Comment:
- possible security vulnerability if users forget to specify the environment variable
- can it default to a derivation `superuser_id` for example?
  - this way the secret is unique and the same after restarts and also can be recovered from the DB

Reply:
you will need to help me here, that is over my head. keep in mind superuser can change